cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -154,7 +154,11 @@ function wait-for-jobs() {
     local rv=0
     local pid
     for pid in $(jobs -p); do
-        wait ${pid} || (rv=1; log ERR "error in pid ${pid}")
```

Comment:
Thanks for the hint.
I'll keep it as it is now (with the 'if'), because I tested it and it works.